djangosaml2idp/views.py

```diff
@@ -128,6 +165,10 @@ def get(self, request, *args, **kwargs):
             sp_config = settings.SAML_IDP_SPCONFIG[resp_args['sp_entity_id']]
         except Exception:
             return self.handle_error(request, exception=ImproperlyConfigured("No config for SP %s defined in SAML_IDP_SPCONFIG" % resp_args['sp_entity_id']), status=400)
+        user_identity = getattr(
```

Comment:
Is there a reason this is repeated here, and cannot use the `user_identity` globally defined on top?